tests/unit/main.c

```diff
@@ -399,20 +400,44 @@ ctest_is_unit_test(const char *argv0)
    return 0;
 }
 
-// Determine the log message verbosity to use for this test run.
-// If env var VERBOSE is unset, or is set to "0" or "false", the be quiet.
-// Otherwise, be verbose.
-static bool
+/*
+ * Determine the log message verbosity level to use for this test run.
+ *
+ * If env var VERBOSE is unset, or is set to "0" or "false", then be quiet.
+ * If env var VERBOSE=1 or "true", be verbose reporting only errors.
+ * Otherwise, settle for the specified verbosity level.
```

Comment:
I think it's ok to break the "false" and "true" usage in service of having a simple numeric representation.

Reply:
Applied this change in latest upload.